index.html

```diff
@@ -330,8 +353,12 @@ <h2 class="main-heading">Grids</h2>
               eiusmod tempor incididunt ut labore et dolore magna aliqua.
             </div>
           </div>
-          <div class="center-heading-float">Above are the Grids, which can be used to build a website. They can be used to divide space for components also. Below is the html code of them.</div>
-          <img class="html-image" src="code/grid-html (2).png"/>
```

Comment:
It would be better if you allow users to copy-paste the code. Tip: you can use `<code></code>` element to display source code. Use `<pre> </pre>` tag to display string as it is without any formatting. and `&lt;` `&gt;` to escape `<` and `>` characters respectively.
e.g.

&lt;pre&gt;
&lt;code&gt;
\&lt; button class="btn-primary"\&gt; Add to Cart \&lt;/button\&gt;
&lt;/code&gt;
&lt;/pre&gt;

output: `<button class=".btn-primary">  Add to Cart</button>`